### ISSUE TITLE
Request: add a second argument to get for default value

### ISSUE THREAD

**Issue by cdarken** [pinguinjkeke/vue-local-storage#10]
It would help to shave a few lines for testing if it's null or not and you have to initialize an array or an object so you can add values to it.

Example:
```js
let storedData = this.$localStorage.get('storedData', {});
```
instead of
```js
let storedData = this.$localStorage.get('storedData');
if (storedData === null) {
    storedData = {}
}
```

**Comment by cdarken** [comment]
Ok, I think I missed something. I forgot there is a configuration phase where you define the properties to be stored and I removed manually the key from Chrome developer tools and then it returned null. The default value is initialized when the component is created, if you delete the key while the app is running, you get the error. Same thing happens if you call `remove` though. I think `remove` should reset the default value when you remove a key.

**Comment by pinguinjkeke** [comment]
Hi @cdarken. It's a great enhancement! I'll implement it ASAP

**Comment by cdarken** [comment]
Hi, @pinguinjkeke. You mean the reset in case it's removed? Because the default behavior works fine for settings default values at init, now that I remembered how to use it :)

**Comment by pinguinjkeke** [comment]
@cdarken default option writes to localStorage on initialization if it's empty. After remove it will be null.
Adding second argument to ```get``` method will give you specified value if ```null``` returned from localStorage

**Comment by pinguinjkeke** [comment]
Feature is implemented in 0.2.0 version
